flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -720,6 +768,88 @@ void testAlterMaterializedTableWithoutSavepointDirConfigured(
                         "Savepoint directory is not configured, can't stop job with savepoint.");
     }
 
+    @Test
+    void testDropMaterializedTable(@InjectClusterClient RestClusterClient<?> restClusterClient)
```

Comment:
Also test the statement `DROP MATERIALIZED TABLE IF EXISTS xxx` if materialized table actually not exists. If the table not exist, we should do nothing.